### PR TITLE
LL-8955 external contributions CI

### PR DESCRIPTION
**Pull request by valpinkman**
Adds a check to CI to run differently if coming from external fork or head repo

### Type

CI

### Context

[LL-8955]

### Parts of the app affected / Test plan

[LL-8955]: https://ledgerhq.atlassian.net/browse/LL-8955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by elbywan — APPROVED** [review]

File: .github/workflows/bundle-app.yml [hunk 1]
```diff
@@ -67,6 +68,7 @@ jobs:
         if: always()
         with:
           node-version: 14.x
+          cache: yarn
```
Comment (elbywan) [inline .github/workflows/bundle-app.yml:71]: Maybe yarn should be installed before this action runs?

**Review by elbywan — CHANGES_REQUESTED** [review]
Wait I think I approved too quickly, I think there is an issue in the order of the steps in the action file 🤭

File: .github/workflows/bundle-app.yml (hunk 1 above)
Comment (valpinkman) [reply .github/workflows/bundle-app.yml:74]: yup

**Review by elbywan — APPROVED** [review]
👌

**Pull request merged**